pkg/pipeline/CdHandler.go

```diff
@@ -964,6 +969,19 @@ func (impl *CdHandlerImpl) getLogsFromRepository(pipelineId int, cdWorkflow *pip
 			CredentialFileJsonData: impl.config.BlobStorageGcpCredentialJson,
 		},
 	}
+
+	useExternalBlobStorage := isExt && !impl.config.UseBlobStorageConfigInCdWorkflow
```

Comment:
add TODO so that we know that needs to be handled in future to persist and get this flag from DB

Reply:
done